data.py

```diff
@@ -42,30 +44,33 @@ def parse(data_json):
   global eateries
   for eatery in data_json['data']['eateries']:
     new_eatery = schema.EateryType(
+        about=eatery['about'],
+        about_short=eatery['aboutshort'],
+        campus_area=parse_campus_area(eatery['campusArea']),
+        coordinates=parse_coordinates(eatery['coordinates']),
+        dining_items=parse_dining_items(eatery['diningItems'], eatery['id']),
+        eatery_type=eatery['eateryTypes'][0]['descr'],
         id=eatery['id'],
-        slug=eatery['slug'],
+        image_url=get_image_url(eatery['slug']),
+        location=eatery['location'],
         name=eatery['name'],
         name_short=eatery['nameshort'],
-        about=eatery['about'],
-        about_short=eatery['aboutshort'],
-        image_url=parse_image_url(eatery['about']),
+        operating_hours=parse_operating_hours(eatery['operatingHours'], eatery['id']),
         payment_methods=parse_payment_methods(eatery['payMethods']),
+        phone=parse_phone(eatery['contactPhone'], eatery['name']),
+        slug=eatery['slug']
         # calender_id=eatery['googleCalenderId'],
-        location=eatery['location'],
-        operating_hours=parse_operating_hours(eatery['operatingHours'], eatery['id']),
-        coordinates=parse_coordinates(eatery['coordinates']),
-        campus_area=parse_campus_area(eatery['campusArea'])
     )
     eateries[new_eatery.id] = new_eatery
 
-def parse_image_url(string):
-  soup = BeautifulSoup(string, 'lxml')
-  found = soup.find('img')
-  if found is not None:
-    image_url = found['src']
-    return image_url
-  print('Image url not found') # Should it be raise(FileNotFound()) exception?
-  return None
+def get_image_url(slug):
+  return constants.ASSET_BASE_URL + 'eatery-images/' + slug + '.jpg'
+
+def parse_phone(string, name):
+  if string is None:
+    print(name + ' missing phone number')
```

Comment:
do we need print statements?

Reply:
no but its nice to see who doesn't have numbers